homeassistant/components/camera/generic.py

```diff
@@ -10,37 +10,37 @@
 from requests.auth import HTTPBasicAuth, HTTPDigestAuth
 import voluptuous as vol
 
-from homeassistant.const import CONF_NAME, CONF_USERNAME, CONF_PASSWORD
+from homeassistant.const import (CONF_NAME, CONF_USERNAME, CONF_PASSWORD)
 from homeassistant.exceptions import TemplateError
-from homeassistant.components.camera import PLATFORM_SCHEMA, Camera
+from homeassistant.components.camera import (PLATFORM_SCHEMA, Camera)
 from homeassistant.helpers import config_validation as cv, template
 
-
 _LOGGER = logging.getLogger(__name__)
 
+BASIC_AUTHENTICATION = 'basic'
```

Comment:
I'd like to see these two authentication methods abstracted to `const.py` - both camera platforms are using the same constants